_co-op-companies/O-Blackberry.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "BlackBerry"
+identification: "blackberry"
+description: "BlackBerry is a Canadian multinational company specializing in enterprise software and the Internet of things."
+link: "https://www.blackberry.com/ca/en"
+image: "assets/img/logos/Blackberry.png"
+width: "100px"
+complete: false
```

Comment:
Having this attribute set to `false` takes the user to the link and doesn't display the modal. Since we have a description for the company, let's set this value to `true` so users can see it 😄